Userland/Libraries/LibWeb/CSS/CSSStyleDeclaration.cpp

```diff
@@ -198,7 +203,11 @@ String CSSStyleDeclaration::get_property_value(StringView property_name) const
     auto property_id = property_id_from_string(property_name);
     if (property_id == CSS::PropertyID::Invalid)
         return {};
-    auto maybe_property = property(property_id);
+    Optional<StyleProperty> maybe_property;
+    if (property_id == CSS::PropertyID::Custom)
+        maybe_property = custom_property(property_name.to_string());
+    else
+        maybe_property = property(property_id);
```

Comment:
```suggestion
    auto maybe_property = property_id == CSS::PropertyID::Custom
        ? custom_property(property_name.to_string())
        : property(property_id);
```

Reply:
I completely forgot about the existence of conditional operators. Thanks.